### ISSUE TITLE
Interface with pyuvdata/pyuvsim

### ISSUE THREAD

**Issue by steven-murray** [steven-murray/21cmSense#11]
There is some support currently for setting up by reading from a pyuvdata object. It would be nice to fully test this, and round it out by enabling reading pyuvsim config files.

**Comment by steven-murray** [comment]
For pyuvdata, it would be useful to use the pyuvdata object itself to provide the UV coverage, then use the standard mechanisms to get the sensitivity.

**Comment by steven-murray** [comment]
Might need to use some kind of aggressive BDA to make it more memory efficient, @jpober suggests.

**Comment by steven-murray** [comment]
To be honest, when I go and try to implement this, I find that I don't think there's much use in accepting eg. pyuvsim configs. The intent of those configs is just so much different from 21cmSense. I think instead we just try to maintain a good configuration for 21cmSense itself.

Thoughts, @mkolopanis ?

**Comment by steven-murray** [comment]
Note that the current code has support at least for creating the `Observatory` from uvdata objects (i.e antenna positions), which is probably the biggest thing that you'd want.

**Comment by mkolopanis** [comment]
The only other thing that would come to mind to me is extracting the total integration time from a pyuvsim config. I don't really think it's a big deal though. Seems reasonable to just leave it at accepting a uvdata object.

**Comment by jpober** [comment]
Yeah, I don't see a ton of value in accepting a pyuvsim config file if it's really totally different.  Accepting a UVData object is still valuable though, as it's nice to ask "I have these observations actually in hand, what's their power spectrum sensitivity?"

**Comment by jpober** [comment]
For one object it's not that useful, I agree.  I imagined taking an entire lst-binned observation and building up a uv coverage from that. My comment at the top of this thread is that you perhaps want to do BDA first, so that you're not dealing with a sample every couple of seconds.
